core/modules/views/plugins/views_plugin_display_block.inc

```diff
@@ -125,15 +134,25 @@ class views_plugin_display_block extends views_plugin_display {
     );
 
     $block_description = strip_tags($this->get_option('block_description'));
+    $block_layout_description = strip_tags($this->get_option('block_layout_description'));
     if (empty($block_description)) {
       $block_description = t('None');
     }
-
+    if (empty($block_layout_description)) {
+      $block_layout_description = t('None');
+    }
+    
     $options['block_description'] = array(
       'category' => 'block',
       'title' => t('Block admin label'),
       'value' => views_ui_truncate($block_description, 24),
     );
+    
+    $options['block_layout_description'] = array(
```

Comment:
These internal keys are a mess (not your fault), perhaps this is an opportunity to change "block_description" to "block_admin_label" and we can make this new key "block_admin_description"? Such a change would require an update hook for existing views.